Tests/SnapshotTestingTests/SnapshotTestingTests.swift

```diff
@@ -676,6 +677,24 @@ final class SnapshotTestingTests: XCTestCase {
     }
     #endif
   }
+    
+  @available(iOS 13.0.0, *)
```

Comment:
```suggestion
  @available(iOS 13.0, macOS 10.15, tvOS 13.0, *)
```

Might need to re-run tests on various platforms.